ydkgen/resolver/bundle_resolver.py

```diff
@@ -81,8 +81,8 @@ class BundleDefinition(object):
 
     def __init__(self, data):
         self._name = data['name'].replace('-', '_')
-        self._version = Version(*tuple(data['version'].split('.')))
-        self._core_version = Version(*tuple(data['core-version'].split('.')))
+        self._version = Version(data['version'])
+        self._core_version = Version(data['core-version'])
```

Comment:
This can potentially break line 106 because of the type change

Reply:
It looks like line 106 is only used by bundle_translator.py to render TEMPLATE. That should end up calling str() on the property.